charts/umbrella/Chart.yaml

```diff
@@ -28,7 +28,7 @@ sources:
   - https://github.com/eclipse-tractusx/tractus-x-umbrella
 
 type: application
-version: 0.11.1
+version: 0.11.2
```

Comment:
I think we had convention of bumping the minor when a new component is add, maybe change to 0.12.0

Reply:
Bumped to 0.12.0